diffrax/_term.py

```diff
@@ -288,7 +289,8 @@ def to_ode(self) -> ODETerm:
 - `vector_field`: A callable representing the vector field. This callable takes three
     arguments `(t, y, args)`. `t` is a scalar representing the integration time. `y` is
     the evolving state of the system. `args` are any static arguments as passed to
-    [`diffrax.diffeqsolve`][].
+    [`diffrax.diffeqsolve`][]. This `vector_field` can be a function that returns a
+    jax array, or returns any lineax `AbstractLinearOperator`.
```

Comment:
Can you add a link to the docs for `lineax.AbstractLinearOperator`?
(Also whilst I'm here, nit jax->JAX.)

Reply:
added link and changed jax